parsec@3.0.2012 +cuda %gcc@11.2.0 build fails: mca/device/cuda/transfer.c:168: multiple definition of `parsec_CUDA_d2h_max_flows'

### Steps to reproduce the issue

Reproducible using:
* `spack@develop` (b32cb5765c598e59f2f91d99807b4331f4eba569 from `Fri Aug 12 23:17:20 2022 -0700`)
* `ecpe4s/ubuntu22.04-runner-x86_64:2022-08-01`
* `%gcc@11.2.0`

Concrete spec: [parsec.spec.json.txt](https://github.com/spack/spack/files/9332540/parsec.spec.json.txt)

Concretization:
```
 -   parsec@3.0.2012%gcc@11.2.0+cuda~debug_verbose~ipo~profile+shared build_type=RelWithDebInfo cuda_arch=80 arch=linux-ubuntu22.04-x86_64
[+]      ^bison@3.8.2%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]          ^diffutils@3.8%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]              ^libiconv@1.16%gcc@11.2.0 libs=shared,static arch=linux-ubuntu22.04-x86_64
[+]          ^m4@1.4.19%gcc@11.2.0+sigsegv patches=9dc5fbd,bfdffa7 arch=linux-ubuntu22.04-x86_64
[+]              ^libsigsegv@2.13%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]          ^perl@5.34.1%gcc@11.2.0+cpanm+shared+threads arch=linux-ubuntu22.04-x86_64
[+]              ^berkeley-db@18.1.40%gcc@11.2.0+cxx~docs+stl patches=b231fcc arch=linux-ubuntu22.04-x86_64
[+]              ^bzip2@1.0.8%gcc@11.2.0~debug~pic+shared arch=linux-ubuntu22.04-x86_64
[+]              ^gdbm@1.19%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                  ^readline@8.1.2%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                      ^ncurses@6.2%gcc@11.2.0~symlinks+termlib abi=none arch=linux-ubuntu22.04-x86_64
[+]                          ^pkgconf@1.8.0%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]              ^zlib@1.2.12%gcc@11.2.0+optimize+pic+shared patches=0d38234 arch=linux-ubuntu22.04-x86_64
[+]      ^cmake@3.23.3%gcc@11.2.0~doc+ncurses+ownlibs~qt build_type=Release arch=linux-ubuntu22.04-x86_64
[+]          ^openssl@1.1.1q%gcc@11.2.0~docs~shared certs=mozilla patches=3fdcf2d arch=linux-ubuntu22.04-x86_64
[+]              ^ca-certificates-mozilla@2022-07-19%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]      ^cuda@11.7.0%gcc@11.2.0~allow-unsupported-compilers~dev arch=linux-ubuntu22.04-x86_64
[+]          ^libxml2@2.9.13%gcc@11.2.0~python arch=linux-ubuntu22.04-x86_64
[+]              ^xz@5.2.5%gcc@11.2.0+pic libs=shared,static arch=linux-ubuntu22.04-x86_64
[+]      ^flex@2.6.3%gcc@11.2.0+lex~nls arch=linux-ubuntu22.04-x86_64
[+]          ^findutils@4.9.0%gcc@11.2.0 patches=440b954 arch=linux-ubuntu22.04-x86_64
[+]      ^hwloc@2.8.0%gcc@11.2.0~cairo~cuda~gl~libudev+libxml2~netloc~nvml~oneapi-level-zero~opencl+pci~rocm+shared arch=linux-ubuntu22.04-x86_64
[+]          ^libpciaccess@0.16%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]              ^libtool@2.4.7%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]              ^util-macros@1.19.3%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]      ^mpich@4.0.2%gcc@11.2.0~argobots~cuda+fortran+hwloc+hydra+libxml2+pci~rocm+romio~slurm~two_level_namespace~vci~verbs~wrapperrpath datatype-engine=auto device=ch4 netmod=ofi pmi=pmi arch=linux-ubuntu22.04-x86_64
[+]          ^libfabric@1.14.1%gcc@11.2.0~debug~kdreg fabrics=rxm,sockets,tcp,udp arch=linux-ubuntu22.04-x86_64
[+]          ^yaksa@0.2%gcc@11.2.0~cuda~rocm arch=linux-ubuntu22.04-x86_64
[+]              ^autoconf@2.69%gcc@11.2.0 patches=35c4492,7793209,a49dd5b arch=linux-ubuntu22.04-x86_64
[+]              ^automake@1.16.5%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]              ^python@3.8.13%gcc@11.2.0+bz2+ctypes+dbm~debug+ensurepip+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tix~tkinter~ucs4+uuid+zlib patches=0d98e93,4c24573,f2fd060 arch=linux-ubuntu22.04-x86_64
[+]                  ^expat@2.4.8%gcc@11.2.0+libbsd arch=linux-ubuntu22.04-x86_64
[+]                      ^libbsd@0.11.5%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                          ^libmd@1.0.4%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                  ^gettext@0.21%gcc@11.2.0+bzip2+curses+git~libunistring+libxml2+tar+xz arch=linux-ubuntu22.04-x86_64
[+]                      ^tar@1.34%gcc@11.2.0 zip=pigz arch=linux-ubuntu22.04-x86_64
[+]                          ^pigz@2.7%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                          ^zstd@1.5.2%gcc@11.2.0+programs compression=none libs=shared,static arch=linux-ubuntu22.04-x86_64
[+]                  ^libffi@3.4.2%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
[+]                  ^sqlite@3.38.5%gcc@11.2.0+column_metadata+dynamic_extensions+fts~functions+rtree arch=linux-ubuntu22.04-x86_64
[+]                  ^util-linux-uuid@2.37.4%gcc@11.2.0 arch=linux-ubuntu22.04-x86_64
```

Install error:
```
$> spack install -f ./parsec.spec.json
...
==> Installing parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe
==> No binary for parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe found: installing from source
==> Fetching https://mirror.spack.io/_source-cache/archive/f5/f565bcfffe106be8237b6aea3e83a5770607b7236606414b6f270244fa6ec3bc.tar.bz2
==> No patches needed for parsec
==> parsec: Executing phase: 'cmake'
==> parsec: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

5 errors found in build log:
     595    cd /tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v/parsec && /spack/lib/spack/env/gcc/gcc -DBUILDING_PARSEC
            -DYYERROR_VERBOSE -D_GNU_SOURCE -Dparsec_EXPORTS -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec/include -I/t
            mp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjy
            qum2fipnpdoe/spack-build-2dfee2v/parsec/include -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v -I/spack/o
            pt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/hwloc-2.8.0-27nq4ccqncmpz3vzybwawofsmtij5clb/include -I/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/cuda-11.7.
            0-a6fzmox2kny2lzgkfkbpsw4axrksf4aj/include -I/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/mpich-4.0.2-gpzf75yhrekurnz3v63fmevfjez2x2q2/include -O2 -g -DNDE
            BUG -fPIC -m64 -Og -g3 -Wall -Wextra -std=gnu11 -MD -MT parsec/CMakeFiles/parsec.dir/parsec_hwloc.c.o -MF CMakeFiles/parsec.dir/parsec_hwloc.c.o.d -o CMakeFiles/pa
            rsec.dir/parsec_hwloc.c.o -c /tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec/parsec_hwloc.c
     596    [100%] Building C object parsec/CMakeFiles/parsec.dir/data_dist/hash_datadist.c.o
     597    cd /tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v/parsec && /spack/lib/spack/env/gcc/gcc -DBUILDING_PARSEC
            -DYYERROR_VERBOSE -D_GNU_SOURCE -Dparsec_EXPORTS -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec/include -I/t
            mp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjy
            qum2fipnpdoe/spack-build-2dfee2v/parsec/include -I/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v -I/spack/o
            pt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/hwloc-2.8.0-27nq4ccqncmpz3vzybwawofsmtij5clb/include -I/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/cuda-11.7.
            0-a6fzmox2kny2lzgkfkbpsw4axrksf4aj/include -I/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/mpich-4.0.2-gpzf75yhrekurnz3v63fmevfjez2x2q2/include -O2 -g -DNDE
            BUG -fPIC -m64 -Og -g3 -Wall -Wextra -std=gnu11 -MD -MT parsec/CMakeFiles/parsec.dir/data_dist/hash_datadist.c.o -MF CMakeFiles/parsec.dir/data_dist/hash_datadist.
            c.o.d -o CMakeFiles/parsec.dir/data_dist/hash_datadist.c.o -c /tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec/d
            ata_dist/hash_datadist.c
     598    [100%] Linking Fortran shared library libparsec.so
     599    cd /tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v/parsec && /spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-1
            1.2.0/cmake-3.23.3-vlcztgknhvh652pvggece6rwshuhxenz/bin/cmake -E cmake_link_script CMakeFiles/parsec.dir/link.txt --verbose=1
     600    /spack/lib/spack/env/gcc/gfortran -fPIC -O2 -g -DNDEBUG -Wl,-rpath -Wl,/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/mpich-4.0.2-gpzf75yhrekurnz3v63fmevfjez
            2x2q2/lib -shared -Wl,-soname,libparsec.so.3 -o libparsec.so.3.0.2012 CMakeFiles/parsec.dir/arena.c.o CMakeFiles/parsec.dir/barrier.c.o CMakeFiles/parsec.dir/bindt
            hread.c.o CMakeFiles/parsec.dir/parsec.c.o CMakeFiles/parsec.dir/data.c.o CMakeFiles/parsec.dir/data_distribution.c.o CMakeFiles/parsec.dir/debug_marks.c.o CMakeFi
            les/parsec.dir/mca/mca_repository.c.o CMakeFiles/parsec.dir/mempool.c.o CMakeFiles/parsec.dir/private_mempool.c.o CMakeFiles/parsec.dir/remote_dep.c.o CMakeFiles/p
            arsec.dir/scheduling.c.o CMakeFiles/parsec.dir/compound.c.o CMakeFiles/parsec.dir/vpmap.c.o CMakeFiles/parsec.dir/maxheap.c.o CMakeFiles/parsec.dir/hbbuffer.c.o CM
            akeFiles/parsec.dir/datarepo.c.o CMakeFiles/parsec.dir/interfaces/interface.c.o CMakeFiles/parsec.dir/interfaces/superscalar/parsec_dtd_data_flush.c.o CMakeFiles/p
            arsec.dir/interfaces/superscalar/overlap_strategies.c.o CMakeFiles/parsec.dir/interfaces/superscalar/insert_function.c.o CMakeFiles/parsec.dir/mca/device/device.c.
            o CMakeFiles/parsec.dir/mca/device/cuda/device_cuda_component.c.o CMakeFiles/parsec.dir/mca/device/cuda/device_cuda_module.c.o CMakeFiles/parsec.dir/mca/device/cud
            a/transfer.c.o CMakeFiles/parsec.dir/mca/pins/pins.c.o CMakeFiles/parsec.dir/mca/pins/pins_init.c.o CMakeFiles/parsec.dir/mca/pins/iterators_checker/pins_iterators
            _checker_component.c.o CMakeFiles/parsec.dir/mca/pins/iterators_checker/pins_iterators_checker_module.c.o CMakeFiles/parsec.dir/mca/pins/print_steals/pins_print_st
            eals_component.c.o CMakeFiles/parsec.dir/mca/pins/print_steals/pins_print_steals_module.c.o CMakeFiles/parsec.dir/mca/pins/ptg_to_dtd/pins_ptg_to_dtd_component.c.o
             CMakeFiles/parsec.dir/mca/pins/ptg_to_dtd/pins_ptg_to_dtd_module.c.o CMakeFiles/parsec.dir/mca/sched/ap/sched_ap_component.c.o CMakeFiles/parsec.dir/mca/sched/ap/
            sched_ap_module.c.o CMakeFiles/parsec.dir/mca/sched/gd/sched_gd_component.c.o CMakeFiles/parsec.dir/mca/sched/gd/sched_gd_module.c.o CMakeFiles/parsec.dir/mca/sche
            d/ip/sched_ip_component.c.o CMakeFiles/parsec.dir/mca/sched/ip/sched_ip_module.c.o CMakeFiles/parsec.dir/mca/sched/lfq/sched_lfq_component.c.o CMakeFiles/parsec.di
            r/mca/sched/lfq/sched_lfq_module.c.o CMakeFiles/parsec.dir/mca/sched/lhq/sched_lhq_component.c.o CMakeFiles/parsec.dir/mca/sched/lhq/sched_lhq_module.c.o CMakeFile
            s/parsec.dir/mca/sched/ll/sched_ll_component.c.o CMakeFiles/parsec.dir/mca/sched/ll/sched_ll_module.c.o CMakeFiles/parsec.dir/mca/sched/ltq/sched_ltq_component.c.o
             CMakeFiles/parsec.dir/mca/sched/ltq/sched_ltq_module.c.o CMakeFiles/parsec.dir/mca/sched/pbq/sched_pbq_component.c.o CMakeFiles/parsec.dir/mca/sched/pbq/sched_pbq
            _module.c.o CMakeFiles/parsec.dir/mca/sched/rnd/sched_rnd_component.c.o CMakeFiles/parsec.dir/mca/sched/rnd/sched_rnd_module.c.o CMakeFiles/parsec.dir/mca/sched/sp
            q/sched_spq_component.c.o CMakeFiles/parsec.dir/mca/sched/spq/sched_spq_module.c.o CMakeFiles/parsec.dir/datatype/datatype_mpi.c.o CMakeFiles/parsec.dir/parsec_hwl
            oc.c.o CMakeFiles/parsec.dir/data_dist/hash_datadist.c.o "CMakeFiles/parsec-base-obj.dir/class/parsec_dequeue.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_fif
            o.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_lifo.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_list.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_o
            bject.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_value_array.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_hash_table.c.o" "CMakeFiles/parsec-base-obj.d
            ir/class/parsec_rwlock.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_future.c.o" "CMakeFiles/parsec-base-obj.dir/class/parsec_datacopy_future.c.o" "CMakeFiles/
            parsec-base-obj.dir/class/info.c.o" "CMakeFiles/parsec-base-obj.dir/utils/argv.c.o" "CMakeFiles/parsec-base-obj.dir/utils/cmd_line.c.o" "CMakeFiles/parsec-base-obj
            .dir/utils/colors.c.o" "CMakeFiles/parsec-base-obj.dir/utils/parsec_environ.c.o" "CMakeFiles/parsec-base-obj.dir/utils/installdirs.c.o" "CMakeFiles/parsec-base-obj
            .dir/utils/keyval_parse.c.o" "CMakeFiles/parsec-base-obj.dir/utils/mca_param.c.o" "CMakeFiles/parsec-base-obj.dir/utils/mca_param_cmd_line.c.o" "CMakeFiles/parsec-
            base-obj.dir/utils/mca_parse_paramfile.c.o" "CMakeFiles/parsec-base-obj.dir/utils/os_path.c.o" "CMakeFiles/parsec-base-obj.dir/utils/output.c.o" "CMakeFiles/parsec
            -base-obj.dir/utils/show_help.c.o" "CMakeFiles/parsec-base-obj.dir/utils/zone_malloc.c.o" "CMakeFiles/parsec-base-obj.dir/utils/atomic_external.c.o" "CMakeFiles/pa
            rsec-base-obj.dir/utils/debug.c.o" "CMakeFiles/parsec-base-obj.dir/utils/win_compat.c.o" "CMakeFiles/parsec-base-obj.dir/show_help_lex.l.c.o" "CMakeFiles/parsec-ba
            se-obj.dir/keyval_lex.l.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/matrix.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/matrixtypes.c.o" "
            data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/map_operator.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/two_dim_tabular.c.o" "data_dist/matrix/CMa
            keFiles/parsec-data-matrix.dir/grid_2Dcyclic.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/two_dim_rectangle_cyclic.c.o" "data_dist/matrix/CMakeFiles/pa
            rsec-data-matrix.dir/two_dim_rectangle_cyclic_band.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/sym_two_dim_rectangle_cyclic.c.o" "data_dist/matrix/CMa
            keFiles/parsec-data-matrix.dir/sym_two_dim_rectangle_cyclic_band.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/vector_two_dim_cyclic.c.o" "data_dist/mat
            rix/CMakeFiles/parsec-data-matrix.dir/reduce_wrapper.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/apply_wrapper.c.o" "data_dist/matrix/CMakeFiles/parse
            c-data-matrix.dir/subtile.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/scalapack_convert.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/reduc
            e_col.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/reduce_row.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/reduce.c.o" "data_dist/matrix/CM
            akeFiles/parsec-data-matrix.dir/diag_band_to_rect.c.o" "data_dist/matrix/CMakeFiles/parsec-data-matrix.dir/apply.c.o" "data_dist/matrix/redistribute/CMakeFiles/par
            sec-data-redistribute.dir/redistribute_wrapper.c.o" "data_dist/matrix/redistribute/CMakeFiles/parsec-data-redistribute.dir/redistribute_dtd.c.o" "data_dist/matrix/
            redistribute/CMakeFiles/parsec-data-redistribute.dir/redistribute.c.o" "data_dist/matrix/redistribute/CMakeFiles/parsec-data-redistribute.dir/redistribute_reshuffl
            e.c.o" fortran/CMakeFiles/parsec_fortran.dir/parsecf.F90.o fortran/CMakeFiles/parsec_fortran.dir/parsecf.c.o  -Wl,-rpath,/spack/opt/spack/linux-ubuntu22.04-x86_64/
            gcc-11.2.0/hwloc-2.8.0-27nq4ccqncmpz3vzybwawofsmtij5clb/lib:/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/mpich-4.0.2-gpzf75yhrekurnz3v63fmevfjez2x2q2/lib::
            :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: -L/usr/lib/gcc/x86_64-linux-gnu/11 -L/usr/lib/x86_64-linux
            -gnu -L/usr/lib -L/lib/x86_64-linux-gnu -L/lib -lgfortran -lm -lgcc_s -lgcc -lquadmath -lm -lgcc_s -lgcc -lc -lgcc_s -lgcc -latomic /spack/opt/spack/linux-ubuntu22
            .04-x86_64/gcc-11.2.0/hwloc-2.8.0-27nq4ccqncmpz3vzybwawofsmtij5clb/lib/libhwloc.so /spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/cuda-11.7.0-a6fzmox2kny2lzg
            kfkbpsw4axrksf4aj/lib64/libcudart_static.a -ldl /usr/lib/x86_64-linux-gnu/librt.a /spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/mpich-4.0.2-gpzf75yhrekurnz3
            v63fmevfjez2x2q2/lib/libmpi.so -lm -lquadmath -lc -latomic /spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/hwloc-2.8.0-27nq4ccqncmpz3vzybwawofsmtij5clb/lib/li
            bhwloc.so /spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/cuda-11.7.0-a6fzmox2kny2lzgkfkbpsw4axrksf4aj/lib64/libcudart_static.a -ldl /usr/lib/x86_64-linux-gnu
            /librt.a
  >> 601    /usr/bin/ld: CMakeFiles/parsec.dir/mca/device/cuda/transfer.c.o:/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec
            /mca/device/cuda/transfer.c:168: multiple definition of `parsec_CUDA_d2h_max_flows'; CMakeFiles/parsec.dir/mca/device/cuda/device_cuda_component.c.o:/tmp/root/spac
            k-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-src/parsec/mca/device/cuda/device_cuda_component.c:43: first defined here
  >> 602    collect2: error: ld returned 1 exit status
  >> 603    make[2]: *** [parsec/CMakeFiles/parsec.dir/build.make:1095: parsec/libparsec.so.3.0.2012] Error 1
     604    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v'
  >> 605    make[1]: *** [CMakeFiles/Makefile2:384: parsec/CMakeFiles/parsec.dir/all] Error 2
     606    make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-parsec-3.0.2012-2dfee2vps3raburrlmjyqum2fipnpdoe/spack-build-2dfee2v'
  >> 607    make: *** [Makefile:169: all] Error 2
```

### Error message

<details><summary>Error message</summary><pre>
... see above
</pre></details>


### Information on your system

* **Spack:** 0.19.0.dev0 (b32cb5765c598e59f2f91d99807b4331f4eba569)
* **Python:** 3.10.4
* **Platform:** linux-ubuntu22.04-zen2
* **Concretizer:** clingo

### Additional information

[spack-build-out.txt](https://github.com/spack/spack/files/9332538/spack-build-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/9332539/spack-build-env.txt)

@abouteiller @bosilca @herault @wspear

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/parsec/package.py
+++ b/var/spack/repos/builtin/packages/parsec/package.py
@@ -26,6 +26,11 @@ class Parsec(CMakePackage, CudaPackage):
 
     version("master", branch="master")
     version(
+        "3.0.2209",
+        sha256="67d383d076991484cb2a265f56420abdea7cc1f329c63ac65a3e96fbfb6cc295",
+        url="https://bitbucket.org/icldistcomp/parsec/get/parsec-3.0.2209.tar.bz2",
+    )
+    version(
         "3.0.2012",
         sha256="f565bcfffe106be8237b6aea3e83a5770607b7236606414b6f270244fa6ec3bc",
         url="https://bitbucket.org/icldistcomp/parsec/get/parsec-3.0.2012.tar.bz2",

